src/theme/index.ts

```diff
@@ -26,7 +26,7 @@ import { getMuiCircularProgressThemeOverrides } from '../ProgressIndicator/Circu
 import { getMuiSnackbarThemeOverrides } from '../Snackbar/Snackbar';
 import { getMuiRadioThemeOverrides } from '../Radio';
 import { getMuiButtonThemeOverrides } from '../Button/Button';
-// import { getMuiToggleButtonThemeOverrides } from '../hidden_components/ToggleButton/ToggleButton'; DXQ-34068 - currently hidden as requested by UIUX
+import { getMuiToggleButtonThemeOverrides } from '../ToggleButton/ToggleButton'; // DXQ-34068 - currently hidden as requested by UIUX
```

Comment:
Hey @kristineapallaHCL, the comment looks wrong to me.

Reply:
right, I forgot to remove the comment